### ISSUE TITLE
[BUG] abnormally high value transfer in `tokens_ethereum.transfers`

### ISSUE THREAD

**Issue by jeff-dude** [duneanalytics/spellbook#6475]
### Description
- transfer happened on alchemix finance
- appears to be an internal migration of tokens across vaults
- while this value was technically transferred, it blows up the scale of the values in the `amount_usd` column
- https://etherscan.io/tx/0xb23aaecc086996f8059a0de67819b0e45d82e68c2581edf889b3177d7aa96ea6
```
select
  *
from
  tokens_ethereum.transfers
where
  tx_to = 0x303241e2b3b4aed0bb0f8623e7442368fed8faf3
  and tx_hash = 0xb23aaecc086996f8059a0de67819b0e45d82e68c2581edf889b3177d7aa96ea6
  and block_date > now() - interval '90' day
```

### Current behavior
please see description above.

### Expected behavior
introduce more opinion on these datasets and filter out transfers without true economic activity onchain.

### Impacted model(s)
```
tokens_ethereum.transfers
```

### Possible solution
we should look to filter out this tx from the dataset. however, rather than hardcoding the tx, an ideal solution may be to write a subquery to find tx's that fit this description and filter out for this tx and all potential future ones. for the subquery, think about looking for transfers with value > an amount, maybe 100 billion? even that is high, but a start.

with transfers being built in macros, this should apply cross-chain as well. in order to avoid full refresh on all chains, we can remove modified step in prod orchestration and only run affected chains (ethereum).

**Comment by jeff-dude** [comment]
@Hosuke this can be considered lower priority vs. the `sei` dex additions. once that is complete, we can look to revisit this one.

**Comment by jeff-dude** [comment]
@0xBoxer what do you think about the arbitrary value of ~100 billion for the subquery to filter out tx's that aren't real economic activity onchain? would you rather set it higher or lower?

**Comment by jeff-dude** [comment]
a finding while looking into this bug. we should create a separate PR to remove bad pricing feeds on ethereum:
- MODA: eth:0x1117aC6Ad6Cdf1A3BC543baD3B133724620522d5
- IDEX: eth:0xb705268213d593b8fd88d3fdeff93aff5cbdcfae
- TIME: bnb:0x26619FA1d4c957C58096bBbeCa6588dCFB12E109
- EGX: polygon:0x8Db0A6d1b06950b4e81c4F67D1289Fc7B9359c7f
- USDEX: bnb:0x829c09fCc46D9fd31967272ABA245BEF9f727F93
- LUNC: eth:0xd2877702675e6cEb975b4A1dFf9fb7BAF4C91ea9
- VERI: eth:0x8f3470a7388c05ee4e7af3d01d8c722b0ff52374
- Coval: eth:0x3D658390460295FB963f54dC0899cfb1c30776Df
- MTL: eth:0xF433089366899D83a9f26A773D59ec7eCF30355e
- HEX: eth:0x2b591e99afE9f32eAA6214f7B7629768c40Eeb39
- ZIL: bnb:0xb86AbCb37C3A4B64f74f59301AFF131a1BEcC787
- SEELE: eth:0xb1e93236ab6073fdac58ada5564897177d4bcc43
- FUND: eth:0xe9b076b476d8865cdf79d1cf7df420ee397a7f75
- CHEX: eth:0x9ce84f6a69986a83d92c324df10bc8e64771030f
